frontend/hooks/useChainDetails.ts

```diff
@@ -0,0 +1,16 @@
+import { CHAIN_CONFIG } from '@/config/chains';
+import { EvmChainId } from '@/enums/Chain';
+
+export const useChainDetails = (chainId: EvmChainId) => {
```

Comment:
don't think we need it as a hook, can just be a fn, but the question above is still relevant, don't we overcomplicating with this?